### PR TITLE
Enable touch events for the stage and audio trimmers

### PR DESCRIPTION
**Pull request by paulkaplan**
Make the audio trimmers and stage work with touch events!

In addition to getting the trimmers and sprite dragging on the stage to work, this also makes all of the related mouse blocks work, so also fixes https://github.com/LLK/scratch-gui/issues/591. 

touch-events

### CODE REVIEW

**Comment by Kenny2github** [comment]
This seems great! I am currently on an iPad so this is good for that too.

It’s a bit concerning that Travis failed though...

**Comment by paulkaplan** [comment]
@Kenny2github woops that is just a failing snapshot test, fixed in 0332606.

File: src/containers/stage.jsx [hunk 1]
```diff
@@ -50,12 +51,18 @@ class Stage extends React.Component {
...
+        document.addEventListener('touchmove', this.onMouseMove);
+        document.addEventListener('touchend', this.onMouseUp);
         canvas.addEventListener('mousedown', this.onMouseDown);
+        canvas.addEventListener('touchstart', this.onMouseDown);
     }
     detachMouseEvents (canvas) {
         document.removeEventListener('mousemove', this.onMouseMove);
         document.removeEventListener('mouseup', this.onMouseUp);
+        document.removeEventListener('touchmove', this.onMouseMove);
+        document.removeEventListener('touchend', this.onMouseEnd);
```
Comment (rschamp) [inline src/containers/stage.jsx]: Looks like a typo here, should be `this.onMouseUp`?

Comment (paulkaplan) [inline src/containers/stage.jsx]: Thanks, fixed 	05f550a

**Pull request merged**